src/qt/bitcoingui.cpp

```diff
@@ -399,10 +399,7 @@ void BitcoinGUI::createActions()
             for (const std::pair<const std::string, bool>& i : m_wallet_controller->listWalletDir()) {
                 const std::string& path = i.first;
                 QString name = path.empty() ? QString("["+tr("default wallet")+"]") : QString::fromStdString(path);
-                // Menu items remove single &. Single & are shown when && is in
-                // the string, but only the first occurrence. So replace only
-                // the first & with &&.
```

Comment:
I think that the comment should be adjusted rather than being deleted entirely.

Reply:
Could you suggest wording? English is not my native language

Reply:
Maybe:
```C++
                // Menu items remove single &. Single & are shown when && is in
                // the string. So replace & with &&.
```
?

Reply:
I agree with @hebasto. Small nit:
```
                // Menu items remove single &. Single & is shown when && are in
                // the string. So replace & with &&.
```